### ISSUE TITLE
[BUG] no-skip-heading-levels: Unexpected skipping heading level. <h4> is expected

### ISSUE THREAD

**Issue by tiagoporto** [yeonjuan/html-eslint#440]
**Describe the bug**
Follow the HTML

```html
<div>
  <h1>Heading 1</h1>
  <h2>Heading 2</h2>
  <h3>Heading 3</h3>
  <h4>Heading 4</h4>
  <h5>Heading 5</h5>
  <h6>Heading 6</h6>
</div>
```

The plugin return the followin  error

```
Unexpected skipping heading level. <h4> is expectedeslint[@html-eslint/no-skip-heading-levels](https://html-eslint.org/docs/rules/no-skip-heading-levels)
```

<img width="899" height="212" alt="Image" src="https://github.com/user-attachments/assets/f9baf19f-3e81-4b0f-9104-88bfbe94ebf4" />

**To Reproduce**

Run eslint againt the HTML.

VSCode and cli returns the same error.

**Expected behavior**
No error.

**Additional context**

Version
```
"@html-eslint/eslint-plugin": "^0.47.0",
"@html-eslint/parser": "^0.47.0",
```

**Comment by yeonjuan** [comment]
@tiagoporto Thanks for the report! Let me fix it
